src/main/scala/io/citrine/lolo/validation/CrossValidation.scala

```diff
@@ -7,7 +7,7 @@ import scala.util.Random
 /**
   * Methods tha use cross-validation to calculate predicted-vs-actual data and metric estimates
   */
-object CrossValidation {
+case class CrossValidation(rng: Random = Random) {
```

Comment:
An alternative would be to put `rng` in the method signatures, which would let this stay as an object (basically let the methods be static)..

Reply:
Do you have a preference? I went this route so that the signatures would stay the same, before I realized this would still be a breaking change.

Reply:
I don't think it's breaking to add a new (optional) parameter to the `CrossValidation.kFoldPvA` method. And it feels less disruptive than changing an object to a case class.

Reply:
Fair enough. Reverted.